tailwind.config.js

```diff
@@ -0,0 +1,289 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+	content: ['./src/**/*.{html,js,md,mdx,svelte,ts}'],
+	plugins: [require('@tailwindcss/typography')],
+	theme: {
```

Comment:
This styling is currently borrowed from my personal website, because I wanted something that didn't look terrible without having to think too hard. It should be replaced if/when we have input from a designer